pkg/controller/proxyconfig/controller.go

```diff
@@ -57,6 +75,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	return nil
 }
 
+// handleConfigMap returns true if meta name is "user-ca-bundle" and
+// meta namespace is "openshift-config-managed".
```

Comment:
comment looks wrong.  the configmap name in "openshift-config-managed" is `trusted-ca-bundle`

Reply:
@bparees I'm confused now. I thought `proxy.spec.trustedCA` points to the user-provided configmap which resides in ns `openshift-config`.The controller merges the user-provided ca bundle with the system  contrbundle and spits out the merged bundle into configmap 'trusted-ca'bundle' in ns 'openshift-config-managed'.

Reply:
> spits out the merged bundle into configmap 'trusted-ca'bundle' in ns 'openshift-config-managed'.

right.  Hence my statement of:

> the configmap name in "openshift-config-managed" is trusted-ca-bundle

vs your doc which says:

> handleConfigMap returns true if meta name is "user-ca-bundle" and meta namespace is "openshift-config-managed".

Reply:
godoc for `handleConfigMap()` fixed to reference 'openshift-config' namespace. See https://github.com/openshift/cluster-network-operator/pull/271#discussion_r313666739 for details why a 'openshift-config-managed/trusted-ca-bundle'  watch is not included in this PR.